packaging/rpm/microshift.spec

```diff
@@ -163,6 +163,16 @@ The microshift-multus-release-info package provides release information files fo
 release. These files contain the list of container image references used by
 the Multus CNI for MicroShift and can be used to embed those images into osbuilder blueprints.
 
+%package low-latency
+Summary: Baseline configuration for running low latency workload on MicroShift
+BuildArch: noarch
+Requires: microshift = %{version}
```

Comment:
Do we want kernel-rt dependency here as well?

Reply:
No, kernel-rt is not necessary for low latency workloads